### DISCUSSION TITLE
V1.2 Walkthrough

### DISCUSSION THREAD

**Discussion by Hrishikesh007788** [sinaptik-ai/pandas-ai#556] [Q&A] [answered]
@gventuri Please make a YT video demonstrating additional features of V1.2 especially SQL connectors.

**Comment by gventuri** [comment, marked as answer]
@Hrishikesh007788 sure thing, it's coming asap (probably tomorrow)!

**Reply by Hrishikesh007788** [reply]
Hey, is it out yet?

**Reply by gventuri** [reply]
For now a video is available about connectors: https://www.loom.com/share/db24dea5a9e0428b87ad86ff596d5f7c

More is coming in the coming days!
